<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="profileNameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="profileComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>650</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>testinappi</string>
    </property>
   </widget>
   <widget class="QSlider" name="frequencySlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="frequencySpinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuProfiles">
    <property name="title">
     <string>P&amp;rofiles</string>
    </property>
    <addaction name="actionEdit_current_profile"/>
    <addaction name="actionAdd_new_profile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProfiles"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionEdit_current_profile">
   <property name="text">
    <string>&amp;Edit current profile</string>
   </property>
  </action>
  <action name="actionAdd_profile_from_file">
   <property name="text">
    <string>Add profile from file...</string>
   </property>
  </action>
  <action name="actionAdd_new_profile">
   <property name="text">
    <string>&amp;Add new profile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
